src/managers/quorum.py

```diff
@@ -197,17 +197,21 @@ def update_acls(self, event: RelationEvent | None = None) -> None:
         for client in self.state.clients:
             if not client.database:
                 continue
-
-            generated_acl = make_acl(
-                scheme="sasl",
-                credential=client.username,
-                read="r" in client.extra_user_roles,
-                write="w" in client.extra_user_roles,
-                create="c" in client.extra_user_roles,
-                delete="d" in client.extra_user_roles,
-                admin="a" in client.extra_user_roles,
+            acls = {
+                "read": "r" in client.extra_user_roles,
+                "write": "w" in client.extra_user_roles,
+                "create": "c" in client.extra_user_roles,
+                "delete": "d" in client.extra_user_roles,
+                "admin": "a" in client.extra_user_roles,
+            }
+
+            sasl_acl = make_acl(scheme="sasl", credential=client.username, **acls)
+            digest_acl = make_acl(
+                scheme="digest",
+                credential=make_digest_acl_credential(client.username, client.password),
+                **acls,
             )
-            logger.info(f"{generated_acl=}")
+            logger.info(f"{sasl_acl=}")
```

Comment:
**todo:** Can we drop this to `DEBUG` please?

Reply:
Done in  d135227be7